CHANGELOG.md

```diff
@@ -3,13 +3,15 @@
 ## 3.2.0
 
 ### Features
-- Add a predefined TimeRange filter if there is at least one DateTime* column
```

Comment:
These features have been announced earlier. see  https://github.com/grafana/clickhouse-datasource/blob/main/CHANGELOG.md#210

Reply:
Ah ok, I wasn't sure if it was announced in v2 if it should be in v3. Makes sense, thanks!